easybuild/easyconfigs/r/R-bundle-CRAN/R-bundle-CRAN-2023.12-foss-2023a.eb

```diff
@@ -3430,6 +3430,12 @@ exts_list = [
     ('missMDA', '1.19', {
         'checksums': ['f9675884829b2fef75237c335b21991d163674320e766523c71c7a853f95e65c'],
     }),
+    ('gcmr', '1.0.3', {
```

Comment:
@pavelToman why is `gcmr` added here and not to the foss-2023b one?

Reply:
add gcmr to 2023a bundle as dependency of scNanoGPS for https://github.com/vscentrum/vsc-software-stack/issues/467
Should I add it either to 2023b? I need it for 2023a

Reply:
yes, better to add it to 2023b too